evm/ts/scripts/contract_governance.ts

```diff
@@ -1,120 +1,107 @@
-  const chainId = Number(process.env.SOURCE_CHAIN_ID!);
-  const emitterChain = Number(process.env.TARGET_CHAIN_ID!);
-  const emitterAddress = Buffer.from(
-    tryNativeToUint8Array(
-      process.env.TARGET_CIRCLE_INTEGRATION_ADDRESS!,
-      "avalanche"
-    )
-  );
-  const domain = Number(process.env.TARGET_DOMAIN!);
-
-  // create unsigned registerEmitterAndDomain governance message
-  const published = governance.publishCircleIntegrationRegisterEmitterAndDomain(
-    timestamp,
-    chainId,
-    emitterChain,
-    emitterAddress,
-    domain
-  );
-
-  // sign the governance VAA with the testnet guardian key
-  const signedMessage = guardians.addSignatures(published, [0]);
-
-  // register the emitter and domain
-  const receipt = await circleIntegration
-    .registerEmitterAndDomain(signedMessage)
-    .then((tx: ethers.ContractTransaction) => tx.wait())
-    .catch((msg: string) => {
-      // should not happen
-      console.log(msg);
-      return null;
-    });
-
-  // check contract state to verify the registration
-  const registeredEmitter = await circleIntegration
-    .getRegisteredEmitter(emitterChain)
-    .then((bytes: ethers.BytesLike) =>
-      Buffer.from(ethers.utils.arrayify(bytes))
+    // MockGuardians and MockCircleAttester objects
+    const guardians = new MockGuardians(await wormhole.getCurrentGuardianSetIndex(), [
+        process.env.TESTNET_GUARDIAN_KEY!,
+    ]);
+
+    // put together VAA
+    const timestamp = getTimeNow();
+    const chainId = Number(process.env.SOURCE_CHAIN_ID!);
+    const emitterChain = Number(process.env.TARGET_CHAIN_ID!);
+    const emitterAddress = Buffer.from(
+        tryNativeToUint8Array(process.env.TARGET_CIRCLE_INTEGRATION_ADDRESS!, "avalanche")
+    );
+    const domain = Number(process.env.TARGET_DOMAIN!);
+
+    // create unsigned registerEmitterAndDomain governance message
+    const published = governance.publishCircleIntegrationRegisterEmitterAndDomain(
+        timestamp,
+        chainId,
+        emitterChain,
+        emitterAddress,
+        domain
     );
-  expect(Buffer.compare(registeredEmitter, emitterAddress)).to.equal(0);
+
+    // sign the governance VAA with the testnet guardian key
+    const signedMessage = guardians.addSignatures(published, [0]);
+
+    // register the emitter and domain
+    const receipt = await circleIntegration
+        .registerEmitterAndDomain(signedMessage)
+        .then((tx: ethers.ContractTransaction) => tx.wait())
+        .catch((msg: string) => {
+            // should not happen
+            console.log(msg);
+            return null;
+        });
+
+    // check contract state to verify the registration
+    const registeredEmitter = await circleIntegration
+        .getRegisteredEmitter(emitterChain)
+        .then((bytes: ethers.BytesLike) => Buffer.from(ethers.utils.arrayify(bytes)));
+    expect(Buffer.compare(registeredEmitter, emitterAddress)).to.equal(0);
 }
 
 async function updateFinality() {
```

Comment:
We can remove this function now.

Reply:
Removed all integration tests here: https://github.com/wormhole-foundation/wormhole-circle-integration/pull/48/commits/6e8d650be73303396d140afe9ad36336a927dfc6. Will add back when we add the Solana Circle Integration program to main

Reply:
Removed here: https://github.com/wormhole-foundation/wormhole-circle-integration/pull/48/commits/1a201f99e8827170ed53559c2869236c81438e2a.